lib/ledger.js

```diff
@@ -399,17 +399,18 @@ async function _sendElectorPool({electorPoolId, maintainerDoc}) {
     capabilityAction: 'write',
     proofValue: 'MOCKPROOF',
     proofPurpose: 'capabilityInvocation',
+    invocationTarget: 'urn:zcap:root:ledger',
```

Comment:
A ledger would never have an address like this -- this is a zcap identifier. This identifier (well, `urn:zcap:root:encodeURIComponent(ledger ID)`) would be used as the ledger's root capability ID, it would not ever be an invocation target. The ledger ID itself would be used in the invocation target field.

We shouldn't put this here -- even as a mock value -- because it will create more confusion for whomever works on changing this mock proof to a real one. Can you find and put the ledger ID (you should be able to find it in the ledger config, wherever that is) here instead? If the ledger ID is auto-generated later or something, then just auto-generate a mock one and put it here.

Reply:
let me know if this method is correct: https://github.com/veres-one/veres-one/pull/44/commits/4d17de37351a3f067d74dc016337b420f9563659